style.css

```diff
@@ -42,4 +48,75 @@ nav {
   margin: 22px 0 0 5px;
   border-left: 1px solid #000000;
   height: 22px;
+}
+
+section {
+  clear: both;
+  height: auto;
+  width: 100%;
+}
+
+.carousel__image-background {
+  position: absolute;
+  margin-top: 15px;
```

Comment:
You need to add space between your `header` and `main` block. But you add space between your picture and the `carousel`. Why did you add this here?